product/scanning.py

```diff
@@ -21,6 +22,19 @@
 from .model import ContainerImage, Component, UploadResult, UploadStatus
 
 
+class ProcessingMode(Enum):
+    UPLOAD_IF_CHANGED = 1
+    RESCAN = 2
+    FORCE_UPLOAD = 4
+
+
+class AppStatus(Enum):
+    UPLOADED = 1
```

Comment:
_brief_ comments / pydoc strings might be helpful (since we are currently in the course of refactoring as far as the commits I have seen go)